tox.ini

```diff
@@ -311,6 +310,7 @@ passenv =
 setenv =
     TWINE_REPOSITORY_URL = https://test.pypi.org/legacy/
```

Comment:
Actually should we remove this as well? Just to remove confusion about how we're specifying the repository.